srcpkgs/x86info/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'x86info'
+pkgname=x86info
+version=1.30
+revision=1
+only_for_archs="x86_64 x86_64-musl i686 i686-musl"
+build_style=gnu-makefile
+hostmakedepends="python3"
+makedepends="pciutils-devel"
+short_desc="A CPU identification utility"
+maintainer="Foxlet <foxlet@furcode.co>"
+homepage="http://codemonkey.org.uk/projects/x86info/"
+license="GPL-2.0-only"
+distfiles="http://codemonkey.org.uk/projects/x86info/x86info-${version}.tgz"
+checksum=fe3009850e93a9c5fa38ed6bf74d72686043329fead8f1b2be4b5943d4fc9428
+
+case "$XBPS_TARGET_MACHINE" in
+	*-musl) CFLAGS="-D_GNU_SOURCE";;
+esac
+
+do_install() {
+	vinstall x86info 775 usr/bin/x86info
+	vinstall lsmsr 775 usr/bin/lsmsr
```

Comment:
````
vinstall x86info 775 usr/bin
vinstall lsmsr 775 usr/bin
````

Reply:
wouldn't using `vbin x86info` and `vbin lsmsr` be nicer?

Reply:
`vbin` installs file into usr/bin with permissions 755.

Reply:
@foxlet perhaps you could explain why you want 775 instead of 755 ?